crowdnotifier-sdk/src/main/java/org/crowdnotifier/android/sdk/CrowdNotifier.java

```diff
@@ -63,15 +86,25 @@ public static boolean updateCheckIn(long id, long arrivalTime, long departureTim
 		return newExposureEvents;
 	}
 
+	/**
+	 * @return a list of all {@link ExposureEvent} objects that have previously matched with a provided ProblematicEventInfo in the
+	 * checkForMatches function and have not been removed yet.
+	 */
 	public static List<ExposureEvent> getExposureEvents(Context context) {
 		return ExposureStorage.getInstance(context).getEntries();
 	}
 
+	/**
+	 * Deletes all ExposureEvents that are older than maxDaysToKeep days.
+	 */
 	public static void cleanUpOldData(Context context, int maxDaysToKeep) {
 		VenueVisitStorage.getInstance(context).removeEntriesBefore(maxDaysToKeep);
 		ExposureStorage.getInstance(context).removeEntriesBefore(maxDaysToKeep);
 	}
 
+	/**
+	 * Removes an ExposureEvent with the given exposureId.
+	 */
```

Comment:
```suggestion
	 * @param exposureId to be deleted
	 */
```

What happens if the exposureId doesn't exist? Is it ignored?